src/pymovements/datasets/bsc.py

```diff
@@ -175,12 +158,6 @@ class BSC(DatasetDefinition):
         ],
     )
 
-    time_column: str = 'time'
```

Comment:
whoops, were these lines removed accidentally?

Reply:
ah sorry, I somehow thought this was in the `DatasetDefinition` base class. nevermind, this dataset has no gaze files thus no time_column.